themes/default/content/docs/troubleshooting/faq.md

```diff
@@ -1,81 +1,75 @@
-
-When you run a preview, update or destroy, pulumi decrypts this data. It is plain text during the execution of your deployment, and any part of your Pulumi program may access it using the Pulumi config object. To learn more, see [Configuration]({{< relref "/docs/intro/concepts/config" >}}).
+To accomplish a _manual_ rollback after a failed deployment, revert the code and configuration changes of the failed deployment and run `pulumi up` to update your infrastructure to its previous state.
 
-## Are my secrets ever visible?
+There is a [GitHub issue related to the idea of automatic rollbacks](https://github.com/pulumi/pulumi/issues/96) if you would like to add to the discussion.
 
-As noted above, Pulumi provides primitives so you can enforce your secrets are stored in a secure manner in the CLI UI, State file and Pulumi Web Console. During an update, your secrets will be unencrypted in memory and visible to your Pulumi program. It is your responsibility to ensure that you do not persist them outside of Pulumi without securing them. To learn more, see [Secrets]({{< relref "/docs/intro/concepts/secrets#secrets" >}}).
+## Organizations
 
-## How do I create a stack inside an Organization instead of my User account?
+### How do I create a stack inside an Organization instead of my User account?
 
-To create a stack in a different Pulumi organization, simply prefix the stack's
-name with the organization's login. For example:
+To create a stack in a different Pulumi organization, prefix the stack's
+name with the organization name. For example:
 
 ```sh
 $ pulumi stack init acme-corp/widget-server
 ```
 
-## How does Pulumi depend on pulumi.com?
+## Pulumi Service
 
-Pulumi uses pulumi.com to store information about the current state of your application, which is used during updates, previews and destroys as the source of truth for the current state of your cloud resources. We refer to this state as the "checkpoint" for your application. In addition, pulumi.com ensures that for a given stack, only a single update is running at once (so, if you and someone else are collaborating on a stack together, pulumi.com ensures that you both don't update the same stack at the same time.) Once your stack has been deployed, it has no dependency on pulumi.com. To learn more about how the Pulumi engine uses pulumi.com, see [How Pulumi Works]({{< relref "/docs/intro/concepts/how-pulumi-works" >}}).
+### How does Pulumi store state?
 
-## What happens if pulumi.com is down?
+Pulumi needs to store the result of operations. On creation of a Pulumi resource, Pulumi makes a call to the cloud providers’ API and then it stores the result of that API call. The place where Pulumi stores that result is called the “state”. The state can be stored using the Pulumi service, Amazon S3, Azure Blob Storage, or Google Cloud Storage Buckets.
```

Comment:
Because we also refer to it as a checkpoint: 

```suggestion
Pulumi needs to store the result of operations. On creation of a Pulumi resource, Pulumi makes a call to the cloud provider's API and then it stores the result of that API call. The place where Pulumi stores that result is called the "state" or "checkpoint". The state can be stored using the Pulumi Service, Amazon S3, Azure Blob Storage, Google Cloud Storage Buckets, or as a file on your local machine that you manage yourself.
```